src/__tests__/components/header.spec.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Header } from '../../components/header/Index';
+
+describe('Test Header component', () => {
```

Comment:
@kleva-j there's no need to add the word `Test` here and every other place it is present.

Reply:
Working on it.